File: sample/src/main/java/com/adevinta/android/barista/sample/WrappedViewActivity.java
package com.adevinta.android.barista.sample;

import android.app.Activity;
import android.os.Bundle;
import android.view.View;

public class WrappedViewActivity extends Activity {

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_wrapped_view);

    findViewById(R.id.button).setOnClickListener(new View.OnClickListener() {
      @Override
      public void onClick(View v) {
        findViewById(R.id.text).setVisibility(View.VISIBLE);
      }
    });
  }
}

File: sample/src/main/java/com/schibsted/spain/barista/sample/WrappedViewActivity.kt
package com.schibsted.spain.barista.sample

import android.app.Activity
import android.os.Bundle
import android.view.View

class WrappedViewActivity : Activity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_wrapped_view)
    findViewById<View>(R.id.button).setOnClickListener { findViewById<View>(R.id.text).visibility = View.VISIBLE }
  }
}